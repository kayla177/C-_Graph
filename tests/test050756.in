ENTITY M5BCN4DX name_M5BC institution
ENTITY 1Y3OWQGG name_1Y3O author
ENTITY 2K9AUHDS name_2K9A paper
RELATIONSHIP M5BCN4DX cited_by 1Y3OWQGG 5.2
PRINT M5BCN4DX
HIGHEST
EXIT
