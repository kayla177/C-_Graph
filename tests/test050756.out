success
success
success
success
1Y3OWQGG
M5BCN4DX 1Y3OWQGG 5.2
